docs/ref/settings.txt

```diff
@@ -2007,7 +2007,7 @@ Default: ``''`` (Empty string)
 
 A secret key for a particular Django installation. This is used to provide
 :doc:`cryptographic signing </topics/signing>`, and should be set to a unique,
-unpredictable value.
+unpredictable value and valid unicode string.
```

Comment:
What about this one?
A secret key for a particular Django installation. This is used to provide
:doc:`cryptographic signing </topics/signing>`, and should be set to a unique,
unpredictable value. The value should be an Unicode string.